### PR TITLE
Day 2

### PR DESCRIPTION
**Pull request by storyd**
You might like this one, or facepalm when you see it.

### CODE REVIEW

**Review by tmthyjames — COMMENTED** [review]
ok how do i have 98 lines of code and you have like 20???

File: day/2/gold.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# B Z =(C)= 3+6 == 9
+# C X =(B)= 2+0 == 2
+# C Y =(C)= 3+3 == 6
+# C Z =(A)= 1+6 == 7
+
+scores=("B X" "C X" "A X" "A Y" "B Y" "C Y" "C Z" "A Z" "B Z")  # Only a slight reording needed
+
+score=0
+
+while IFS='$\n' read -r round; do
```
Comment (tmthyjames) [inline day/2/gold.sh:23]: i don't even know what this says lol

**Comment by tmthyjames** [comment]
haha this is impressive

**Review by tmthyjames — APPROVED** [review]
LGTM = looks godly to me

File: day/2/gold.sh (hunk 1 above)
Comment (storyd) [reply day/2/gold.sh:23]: `while <condition>` where `<condition>` is
`set the Internal Field Separator to newline for next command` and `read ignoring back slashes each field into variable round` which will return a falsy when it runs out of fields (lines)

**Pull request merged**